.github/workflows/regression.yml

```diff
@@ -28,40 +28,92 @@ jobs:
           sudo ninja -C build install
           cd ..
 
-      - name: Download SVG Regression finder
+      - name: Download SVG Regression finder and setup settings
         run: |
-          wget https://github.com/qarmin/SVG-regression-finder/releases/download/0.2.0/svg_tester
+          wget https://github.com/qarmin/SVG-regression-finder/releases/download/0.3.0/svg_tester
```

Comment:
What do you think if it has a "latest" folder instead of the specific verison? We don't need to update thorvg everytime.